client/src/components/formHelpers/FormFields.js

```diff
@@ -49,14 +49,12 @@ function FormFields(props) {
           const type = name in types ? types[name] : 'text';
           return (
             <div className='inline-form-field' key={key}>
-              <Col sm={3} xs={12}>
+              <Col sm={12} xs={12}>
                 {type === 'hidden' ? null : (
                   <ControlLabel htmlFor={key}>{startCase(name)}</ControlLabel>
                 )}
-              </Col>
-              <Col sm={9} xs={12}>
                 <FormControl
-                  bsSize='lg'
+                  bsSize='md'
```

Comment:
Warning: Failed prop type: Invalid prop `bsSize` of value `md` supplied to `FormControl`, expected one of ["sm","small","lg","large"].

Reply:
This is what I hate about the custom version of `react-bootstrap`, it literally has tons of things missing from its parent bootstrap.